app/models/onet_code.rb

```diff
@@ -0,0 +1,3 @@
+class OnetCode < ApplicationRecord
+  validates :name, :code, presence: true
```

Comment:
Let's add a uniqueness validation to this too. How about also on the database level.